inc/reception.class.php

```diff
@@ -517,9 +521,14 @@ function getForbiddenStandardMassiveAction() {
    function getSpecificMassiveActions($checkitem = null) {
       $isadmin = static::canUpdate();
       $actions = parent::getSpecificMassiveActions($checkitem);
+
+      //remove native transfer action
+      unset($actions[MassiveAction::class.MassiveAction::CLASS_ACTION_SEPARATOR.'add_transfer_list']);
       $sep     = __CLASS__.MassiveAction::CLASS_ACTION_SEPARATOR;
 
       $actions[$sep.'reception'] = __("Take item delivery", "order");
+      $actions[$sep.'transfer_order_item'] = "<i class='fa-fw fas fa-level-up-alt'></i>" .
```

Comment:
1. Transfer should be proposed only corresponding order `is_recursive`.

2. Unless I am wrong, transfer is done directly, without using the transfer list (queue), so `Add to transfer list` should probably replaced by `Transfer`.

Reply:
From ```getSpecificMassiveActions``` I cannot check the ```is_recursive``` value of the original ```order```

Use of ```Transfer``` action is impossible here (see description)